app/src/main/java/com/gdgistanbul/attendence/ui/event_list/EventListFragment.kt

```diff
@@ -22,11 +33,63 @@ class EventListFragment : Fragment() {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        initSearchView()
+        initTabLayout()
+        initRecyclerView()
+        observe(eventListViewModel.memberLiveData, ::showGreeting)
+        observe(eventListViewModel.eventsLiveData, ::showEventList)
+        observeEvent(eventListViewModel.toastLiveData, ::toast)
+    }
+
+    private fun initSearchView() = with(searchView) {
+        measure(
+            FrameLayout.LayoutParams.MATCH_PARENT,
+            FrameLayout.LayoutParams.WRAP_CONTENT
+        )
+
+        layoutParams = (layoutParams as ViewGroup.MarginLayoutParams).apply {
+            topMargin = -(measuredHeight / 2)
+        }
 
-        buttonNavigateUserList.onClick {
-            // TODO - EventID Will replace with returning eventID from MeetupAPI
-            val eventID = (0..99999999).random()
-            navigate(EventListFragmentDirections.actionEventListFragmentToUserListFragment(eventID))
+        collapsingToolbarLayout.minimumHeight = measuredHeight / 2
```

Comment:
I think we can hardcode these values

Reply:
I couldn't count on hardcoded values here. It may slip in different devices.